litmus-portal/backend/graphql-server/go.mod

```diff
@@ -4,5 +4,9 @@ go 1.14
 
 require (
 	github.com/99designs/gqlgen v0.11.3
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/google/uuid v1.1.1
+	github.com/gorilla/mux v1.7.4
```

Comment:
Are we using gorilla mux somewhere?

Reply:
Yes for getting the yaml file, gorilla mux makes it easy to use vars in the url.